hack/test-end-to-end.sh

```diff
@@ -235,8 +235,8 @@ DOCKER_REGISTRY=$(osc get --output-version=v1beta1 --template="{{ .portalIP }}:{
 echo "[INFO] Verifying the docker-registry is up at ${DOCKER_REGISTRY}"
 wait_for_url_timed "http://${DOCKER_REGISTRY}" "[INFO] Docker registry says: " $((2*TIME_MIN))
 
-docker tag -f centos:centos7 ${DOCKER_REGISTRY}/cached/centos:centos7
-docker push ${DOCKER_REGISTRY}/cached/centos:centos7
+docker tag -f openshift/ruby-20-centos7:latest ${DOCKER_REGISTRY}/test/ruby-20-centos7:latest
+docker push ${DOCKER_REGISTRY}/test/ruby-20-centos7:latest
```

Comment:
this needs to be part of the readme instructions too since otherwise your changes break the sample app.

however I think @smarterclayton just delivered some changes that make this unnecessary since he now handles constructing imagerepos with references to dockerhub.